smslibrary/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,10 @@
     <uses-permission android:name="android.permission.RECEIVE_SMS" />
 
     <application>
+        <service
+            android:name=".background.SMSKeepAliveService"
+            android:enabled="true"
+            android:label="SMSKeepAliveService" />
```

Comment:
I thought there was an agreement on removing the service since BroadcastReceivers declared in the manifest can be called by Android at any moment. Maybe it provides some other functionalities that I missed.

Reply:
I really don't get why android refractor delete and build button don't tell me that I forgot to remove things around the project.
My fault indeed.